client/my-sites/stats/features/modules/stats-locations/stats-locations.tsx

```diff
@@ -209,22 +210,9 @@ const StatsLocations: React.FC< StatsModuleLocationsProps > = ( { query, summary
 		</StatsInfoArea>
 	);
 
-	const fakeData = [
```

Comment:
We needed to add coordinates, and I think it’s too much data to keep in this file, so I preferred to move it to its own sample-locations.js file.